ironfish-cli/src/commands/accounts/pay.ts

```diff
@@ -142,23 +131,57 @@ export class Pay extends IronfishCommand {
       }
     }
 
-    if (!from) {
-      const response = await client.getDefaultAccount()
-      const defaultAccount = response.content.account
+    if (!isValidPublicAddress(to)) {
+      this.log(`A valid public address is required`)
+      this.exit(1)
+    }
 
-      if (!defaultAccount) {
-        this.error(
-          `No account is currently active.
-           Use ironfish accounts:create <name> to first create an account`,
-        )
+    if (fee == null) {
+      let suggestedFees = ''
+      try {
+        const response = await client.estimateFee({
+          fromAccountName: from,
+          receives: [
+            {
+              publicAddress: to,
+              amount: CurrencyUtils.encode(amount),
+              memo: memo,
+            },
+          ],
+        })
+
+        switch (flags.priority || 'medium') {
+          case 'low':
+            suggestedFees = CurrencyUtils.renderIron(response.content.low)
+            break
+          case 'high':
+            suggestedFees = CurrencyUtils.renderIron(response.content.high)
+            break
+          default:
+            suggestedFees = CurrencyUtils.renderIron(response.content.medium)
+        }
+      } catch {
+        suggestedFees = ''
       }
 
-      from = defaultAccount.name
+      const input = (await CliUx.ux.prompt(
+        `Enter the fee amount in $IRON (min: ${CurrencyUtils.renderIron(
+          1n,
+        )})\nrecommended: ${suggestedFees}`,
```

Comment:
nit - I think it might look better with min and recommended on the same line:
```
Enter the fee amount in $IRON (min: 0.00000001, recommended: 0.00000460):
```